src/mumble/Audio.h

```diff
@@ -17,6 +17,9 @@
 
 #define SAMPLE_RATE 48000
 
+// interaural delay (in samples) for a sound coming directly from the side of the head
+#define INTERAURAL_DELAY 0.0006 / (1 / static_cast< float >(SAMPLE_RATE))
```

Comment:
We try to define constants as constepxr variables nowadays :)

```suggestion
static constexpr float INTERAURAL_DELAY = 0.0006 / (1 / static_cast< float >(SAMPLE_RATE))
```

Reply:
I don't think `static` has any effect with `constexpr`.

Reply:
Yes I think you are correct.